melodic/distribution.yaml

```diff
@@ -10634,6 +10634,22 @@ repositories:
       url: https://github.com/RoverRobotics/rr_openrover_stack.git
       version: melodic-devel
     status: developed
+  rt_usb_9axisimu_driver:
+    doc:
+      type: git
+      url: https://github.com/rt-net/rt_usb_9axisimu_driver.git
+      version: melodic-devel
+    release:
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/rt-net-gbp/rt_usb_9axisimu_driver-release.git
+      version: 1.0.0-1
+    source:
+      test_pull_requests: true
```

Comment:
@ShotaAk just to double check, have you configured your repository accordingly (see [here](http://wiki.ros.org/buildfarm/Pull%20request%20testing))?

Reply:
Thank you for your notice, I just updated the collaborator access right.

![image](https://user-images.githubusercontent.com/18494952/92422485-8ff9a300-f1b8-11ea-9b18-22ddd60f2050.png)